transformer_engine/common/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@
 cmake_minimum_required(VERSION 3.18)
 
 if(NOT DEFINED CMAKE_CUDA_ARCHITECTURES)
-  set(CMAKE_CUDA_ARCHITECTURES 70 80 90)
+  set(CMAKE_CUDA_ARCHITECTURES 70 75 80 90)
```

Comment:
Turing can be handled by sm_70 and having special versions for it will just increase the library size - can we remove that?